deck.py

```diff
@@ -0,0 +1,68 @@
+from card import Card 
+
+colors = ['Blue', 'Red', 'Yellow','Green']
+abilitys = ['Cancelation', 'Direction', 'Sum_two']
+will_abilitys = ['change_of_color', 'change_of_color+4']
+
+class Deck():
+    def __init__(self):
+        self.decks = []
+        self.decks_compi = []
+
+
+
+    def create_card(self):
```

Comment:
```suggestion
    def create_cards(self):
```
because is creating all the cards